app/controllers/search_controller.rb

```diff
@@ -47,6 +47,8 @@ def get_search_results(query)
         cmr_client.get_variables(query, token)
       when 'services'
         cmr_client.get_services(query, token)
+      when 'tools'
+        cmr_client.get_tools(query, token)
```

Comment:
I can't remember the implementation of Var or S anymore, but I'm pleasantly surprised that this is all you needed to add to the controller.